src/modalCreation.js

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line no-unused-vars
 import getItem, { commentCounter, getComments } from './commentsRequest.js';
 
 const modalDiv = document.getElementById('modalDiv');
```

Comment:
Greta job on adding the counter on the coments count 
![Screenshot 2021-12-24 at 10 01 32](https://user-images.githubusercontent.com/31847346/147337746-24e63db8-bc0f-4c00-8439-eb5ba816b3df.png)